packages/frontend/src/common/components/navigation/Breadcrumbs.tsx

```diff
@@ -0,0 +1,59 @@
+'use client'
+
+import { usePathname } from 'next/navigation'
+import Link from 'next/link'
+
+export function NavigationBreadcrumbs({
```

Comment:
I'll have to look at this one again tomorrow to understand what's going on here :)

Reply:
If the component contains such a significant amount of logic, it should have some tests I guess.

Reply:
Honestly, I just see a lot of complexity and not so many benefits.